RecoTracker/MeasurementDet/plugins/doubleMatch.icc

```diff
@@ -149,12 +149,16 @@ void TkGluedMeasurementDet::doubleMatch(const TrajectoryStateOnSurface& ts, cons
 			    gluedDet, trdir, chelper);
   }
 
-  // if no match found try add mon than try to add stereo...
-//  if (0==collector.size()) 
-//    projectOnGluedDet( collector, monoHits, glbDir);
-//  if (0==collector.size())
-//    projectOnGluedDet( collector, stereoHits, glbDir);
-  
+
+  // auto id = geomDet().geographicalId().subdetId();
+  // if ( (id==3 || id==5) &&  // tob & tib
+  if (ts.globalMomentum().perp2()>0.81f) {
```

Comment:
should something similar be added in the DOUBLE_MATCH false case of TkGluedMeasurementDet::collectRecHits
https://github.com/VinInn/cmssw/blob/d5168db349792cf8631723bdfcb924d55d81e6b9/RecoTracker/MeasurementDet/plugins/TkGluedMeasurementDet.cc#L186
?